config/enabled.yml

```diff
@@ -223,6 +223,10 @@ Style/EmptyLinesAroundAccessModifier:
   Description: "Keep blank lines around access modifiers."
   Enabled: true
 
+Style/EmptyLinesAroundBeginBody:
```

Comment:
No, we don't have the style guide. And we also don't have style guides for other `EmptyLines*` cops.
I'll add the documentation.